docs/compass/01-01-compass.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Overview
+---
+
+>**NOTE:** Compass is a new, experimental component in Kyma. To learn how to enable it, read the [installation](#installation-enable-compass-in-kyma) document.
+
+Compass is a multi-tenant system which consists of components that provide a way to manage your applications across multiple Kyma Runtimes. Using Compass, you can control and monitor your application landscape in one central place.
+
+Compass allows for registering different types of applications and Runtimes.
+These are the types of possible integration levels between an application and Compass:
+- Manual integration - administrator manually provides API or Events metadata to Compass. This type of integration is used mainly for simple use-case scenarios and doesn't support all features.
+- Built-in integration - integration with Compass is built-in inside the application.
+- Proxy - a highly application-specific proxy component provides the integration.
+- Central integration service -  a central service provides integration for the whole group of applications. It manages multiple instances of these applications. You can integrate multiple central services to support different types of applications.
```

Comment:
"configure itself" sounds a bit weird to me...

Reply:
Change applied.